### DISCUSSION TITLE
Webgui has issues when running through Traefik

### DISCUSSION THREAD

**Discussion by cprpolo** [firefly-iii/firefly-iii#9295] [Firefly III]
Hi! :) 
Thanks for the project! 
I have it running on docker via traefik and locally works fine but when going via traefik the webgui seems to break
I have seen it was discussed already: https://github.com/orgs/firefly-iii/discussions/7907
but did not seem like a firefly issue but a traefik issue (although it was with nginx at the time) it also went unanswered. I'd like to bring it up see what i need to do to fix it if possible.
I see the logs in the dev tools:

This is how it looks in firefox:
![image](https://github.com/user-attachments/assets/1c55e492-a1aa-42ca-8e87-c953336f8691)

After trying to login:
![image](https://github.com/user-attachments/assets/45aa18e2-4a7e-4d26-9d60-125cc475eaeb)

I can test things if you need 

Any help? Thanks in advance

**Comment by JC5** [comment]
It looks like your proxy doesn't set the correct headers. Firefly III generates "http" urls.

I don't know how Traefik works, but it's just a fancy script on top of nginx right?

See https://docs.firefly-iii.org/references/faq/install/#how-do-i-set-tls-in-firefly-iii-or-the-data-importer

**Comment by cprpolo** [comment]
I am not super familiar with traefik, i just use it...

I'll check this, for now i'll use it internally, thank you
